h/templates/accounts/notifications.html.jinja2

```diff
@@ -4,5 +4,9 @@
 {% set page_title = 'Notifications' %}
 
 {% block page_content %}
-  {{ form }}
+  {% if user_has_email_address %}
+    {{ form }}
+  {% else %}
+    Oops! We don't have an email address on file, so we can not send you notifications at the moment.
```

Comment:
Alternative message: "Life is boring without notifications, give us your email and we'll send some your way!"

Reply:
IMO a little bit of humor can be good in messages, and we have some precedents (eg. the "Sad highlight" on 5xx error pages). It would be useful to have a style guide about our "voice" though to decide when it is appropriate and when to avoid.

I prefer the message in the diff to the alternative because it explains the problem more clearly. I would remove the "Oops!" though because Hypothesis hasn't experienced an unexpected error. As well as explaining the issue, it would be good to make it clear how to fix it - "To enable notifications, please [set an email address](link to appropriate settings page) for your account". I'm assuming here that the user _can_ set an email address if their account doesn't have one?

Reply:
They can set one, yes (and I agree we should provide a link to the page where they can do so)

Reply:
Yep!  Here's my proposed verbiage (not that anyone asked my opinion!):

"Want notifications? We'll need your email address for that. You can add an email address in your (Account Settings)."

Anyway: action item: please link to the Account Settings page